src/commands/Minion/create.ts

```diff
@@ -51,6 +56,11 @@ export default class extends BotCommand {
 			throw `You don't have the required items to create this item. You need: ${inputItemsString}.`;
 		}
 
+		// Check for any items they cant have 2 of.
+		if (bankHasAllItemsFromBank(userBank, craftableItem.cantHaveItems)) {
```

Comment:
this is a bit flawed as it will only return true if the user has ALL the items from `cantHaveItems`